test/Interop/Cxx/operators/non-member-inline-typechecker.swift

```diff
@@ -21,6 +21,8 @@ let resultEqualEqual = lhs == rhs
 let resultExclaimEqual = lhs != rhs
 let resultLessEqual = lhs <= rhs
 let resultGreaterEqual = lhs >= rhs
+let reusltPlusPlus = lhs += LoadableIntWrapper(value: 1)
```

Comment:
You actually have a `rhs` value that you can use here:
```suggestion
let reusltPlusPlus = lhs += rhs
```

(Same on L25)

Reply:
Wait a second, I didn't read this very clearly. Here you're testing `+=` and `-=` but the operators you added support for are `++` and `--`. So please update both these tests to use `++` and `--` _not_ `+=` and `-=`.

Reply:
Ahh yep, i see thank you!